### PR TITLE
add jailalarm command

### PR DESCRIPTION
**Pull request by bicatcho11**
## Description

add jailalarm command

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ ] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Comment by bicatcho11** [comment]
https://github.com/Qbox-project/qbx_policejob/pull/96

**Comment by mafewtm** [comment]
I'd look at the documentation for the `GetDutyCountType` export and ensure it's being implemented properly

**Comment by bicatcho11** [comment]
Why is there a huge delay in my withdrawal requests?
There are applications that are immediately accepted or rejected
I hope that the team will process my requests quickly
thank you

**Comment by Manason** [comment]
What's the purpose of letting police turn on/off the alarm?

**Comment by bicatcho11** [comment]
The event already exists
I just completed it
prison:client:JailAlarm

File: server/main.lua [hunk 1]
```diff
@@ -119,6 +119,24 @@ exports.ox_inventory:RegisterShop('Canteen', {
...
+    help = locale('info.jail_alarm'),
+    params = {
+        {
+            name = 'type',
+            type = 'string',
+            help = locale('info.jail_alarm')
+        }
+    },
+}, function(source, args)
+    if not exports.qbx_core:GetDutyCountType('leo') then return end
```
Comment (BerkieBb) [inline server/main.lua:132]: GetDutyCountType does not return a boolean or nil at all, it returns a number representing the amount of people on duty for that type

**Review by BerkieBb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**